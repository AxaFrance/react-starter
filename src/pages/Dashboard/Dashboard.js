import React from 'react';

const Dashboard = props => <div>Dashboard</div>;

export default Dashboard;
