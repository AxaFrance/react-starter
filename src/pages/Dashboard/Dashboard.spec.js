import React from 'react';
import { render } from '@testing-library/react';
import Dashboard from './Dashboard';

it('1. Renders Dashboard component without crashing', () => {
  const { asFragment } = render(<Dashboard />);
  expect(asFragment()).toMatchSnapshot();
});
