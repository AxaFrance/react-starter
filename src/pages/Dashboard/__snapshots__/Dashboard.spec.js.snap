// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`1. Renders Dashboard component without crashing 1`] = `
<DocumentFragment>
  <div>
    Dashboard
  </div>
</DocumentFragment>
`;
